website/src/components/header.js

```diff
@@ -1,16 +1,27 @@
 import React, {Component} from 'react';
 import {Link} from 'react-router';
 
+import {LIBRARIES} from '../constants/links';
```

Comment:
This is great for now.

To simplify maintenance when we have this list in several repos, should we load a JSON file from a raw github link (we could store it in the deck.gl repo for now) with the list of links?

We'd use the constant list until we load the live list?